web_monitoring/annotations_import.py

```diff
@@ -0,0 +1,116 @@
+    # Missing step: capture info from the "Who Found This" column and map it to
+    # the author's user record in the database eventually, but that requires a
+    # change to the API
+
+    bool_keys = [
+        'Language alteration',
+        'Content change/addition/removal',
+        'Link change/addition/removal',
+        'Repeated Change across many pages or a domain',
+        'Alteration within sections of a webpage',
+        'Alteration, removal, or addition of entire section(s) of a webpage',
+        'Alteration, removal, or addition of an entire webpage or document',
+        'Overhaul, removal, or addition of an entire website',
+        'Alteration, removal, or addition of datasets'
+    ]
+    string_keys = [
+        'Is this primarily a content or access change (or both)?',
+        'Brief Description',
+        'Topic 1',
+        'Subtopic 1a',
+        'Subtopic 1b',
+        'Topic 2',
+        'Subtopic 2a',
+        'Subtopic 2b',
+        'Topic 3',
+        'Subtopic 3a',
+        'Subtopic 3b',
+        'Any keywords to monitor (e.g. for term analyses)?',
+        'Further Notes',
+        'Ask/tell other working groups?'
+    ]
+    bools_dict = {k: csv_row[k] == '1' for k in bool_keys}
+    strings_dict = {k: csv_row[k] for k in string_keys}
+    annotation = {**bools_dict, **strings_dict}
+
+    significance = 0.0
+    if is_important_changes:
+        importance_significance_mapping = {
+            'low': 0.5,
+            'medium': 0.75,
+            'high': 1.0
+        }
+        row_importance = csv_row['Importance?'].lower().strip()
+        significance = importance_significance_mapping.get(row_importance, 0.0)
+    annotation['significance'] = significance
+
+    try:
+        annotation['priority'] = float(csv_row['Priority (algorithm)'])
+    except:
+        annotation['priority'] = 0.0
+
+    return annotation
+
+def post_annotation(change_ids, annotation):
+    cli = Client.from_env()
+    response = cli.add_annotation(annotation=annotation,
+                                  page_id=change_ids['page_id'],
+                                  to_version_id=change_ids['to_version_id'],
+                                  from_version_id=change_ids['from_version_id'])
+    return response
+
+def main():
+    doc = """Add analyst annotations from a csv file to the Web Monitoring db.
+
+Usage:
+path/to/annotations_import.py <csv_path> [--is_important_changes]
+
+Options:
+--is_important_changes  Was this CSV generated from an Important Changes sheet?
+"""
+    arguments = docopt(doc)
+    is_important_changes = arguments['--is_important_changes']
+    csv_path = arguments['<csv_path>']
+
+    # Missing step: Analyze CSV to determine spreadsheet schema version
+    for row in read_csv(csv_path):
+        change_ids = find_change_ids(row)
+        annotation = create_annotation(row, is_important_changes)
+        response = post_annotation(change_ids, annotation)
+        print(response)
```

Comment:
Logging every line might get a little verbose. You could use a logger and log this at the `DEBUG` level.

Reply:
Done, and including a change to .env.example which actually makes log levels set there stick.